recipes/default.rb

```diff
@@ -0,0 +1,126 @@
+# Copyright:: 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Validate the agent type
+unless %w(google-cloud-ops ops-agent monitoring logging).include? node['final_agent_type']
```

Comment:
What does `google-cloud-ops` type refer to here?

Reply:
This is due to the actual naming of the software requiring `google-cloud-ops` for the file names and service names. The code actually transforms `ops-agent` to `google-cloud-ops`. This is uniform to both Ansible and Puppet projects as well. Instead, the `ops-agent` should be removed from here.